Solutions/README.md

```diff
@@ -1 +1,122 @@
-Guidelines (coming soon)
+# Azure Sentinel Solutions 
+Azure Sentinel Solutions provide an in-product experience for central discoverability, single-step deployment, and enablement of end-to-end product and/or domain and/or vertical scenarios in Azure Sentinel. This experience is powered by Azure Marketplace for Solutions’ discoverability, deployment and enablement and Microsoft Partner Center for Solutions’ authoring and publishing. 
+
+# Why Azure Sentinel Solutions?
+* Customers can easily discover packaged content and integrations that deliver product value or for a domain or for a vertical centrally in Azure Sentinel. 
+* Customers can easily deploy and optionally enable content to get started immediately.
+* Providers or partners can deliver combined product or domain or vertical value via Solutions in Azure Sentinel and be able to productize investments too.
+
+# Solution Package
+Clone the repository [Sentinel-AzMP-Solutions](https://msazure.visualstudio.com/One/_git/Sentinel-AzMP-Solutions) to C:\One.
+## Create Input File
+Create an input file and place it in the path C:\One\Sentinel-AzMP-Solutions\solutions\automation\input.
+
+**Input File Format:**
+```
+/**
+ * Solution Automation Input File Interface
+ * -----------------------------------------------------
+ * The purpose of this interface is to provide detail on
+ *  the various fields the input file can have.
+ */
+interface SolutionAutomationInput {
+  Name: string;                //Solution Name      - Ex. "Symantec Endpoint Protection"
+  Author: string;              //Author of Solution - Ex. "Eli Forbes - v-eliforbes@microsoft.com"
+  Logo: string;                //Link to the Logo used in the CreateUiDefinition.json
+  Description: string;         //Solution Description used in the CreateUiDefinition.json
+  WorkbookDescription: string; //Workbook description from ASI-Portal Workbooks Metadata
+
+  //The following fields take arrays of paths relative to the solutions folder.
+  //Ex. Workbooks: ["Workbooks/SymantecEndpointProtection.json"]
+  Workbooks?: string[];
+  "Analytic Rules"?: string[];
+  Playbooks?: string[];
+  Parsers?: string[];
+  "Hunting Queries"?: string[];
+  "Data Connectors"?: string[];
+  BasePath?: string; //Optional base path to use. Either Internet URL or File Path. Default = "https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/""
```

Comment:
Just a note, I'll be taking out the addition of the solution name to the base path, as it would just cause confusion.